uninit_filter.py

```diff
@@ -15,23 +15,24 @@ def find_uninit_classes(input_file):
     current_cls = None
     uninit_dict = {}
 
-    for file in data.measured_files():
-        with open(file, 'r') as f:
+    for file_name in data.measured_files():
+        with open(file_name, 'r') as f:
             for i, line in enumerate(f):
                 current_line = i + 1
                 if line.startswith('class'):
                     current_cls = re_cls.match(line).group(1)
-                elif re_init.search(line) and (current_line) not in data.lines(file):
-                    if file not in uninit_dict:
-                        uninit_dict[file] = []
-                    uninit_dict[file].append(current_cls)
+                elif re_init.search(line) and (current_line) not in data.lines(file_name):
```

Comment:
no need to have parens around current_line.

Also, does data.lines return a set or a list?  Checking for occurance of something in a set is much more efficient than a list.  If its a list I would convert to a set.

Finally and this is related to the point above, I would make data.lines a variable, up above out of the loop.  You can do the set conversion there.  This will help shorten up the noise in this long if statement and also will be a little faster as you are reducing lookups in the `data` instance dict.